CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+## [2.4.3] - 2023-02-13
```

Comment:
Please remove this line (similar to the other PR)

Reply:
Fixed!